studio/src/pages/dashboard/components/Features.js

```diff
@@ -112,7 +112,7 @@ function Features() {
             actions={[
               <Button
                 onClick={() => {
-                  dispatch(addDefaultFormats(selectedSpace)).then(() => history.push('/advanced/formats'));
+                  dispatch(addDefaultFormats(selectedSpace));
```

Comment:
@pallavimshr please redirect to the formats list page to make it consistent with create policies behaviour.